exonum-java-binding/core/rust/src/utils/mod.rs

```diff
@@ -27,3 +27,20 @@ pub use self::errors::{
     panic_on_exception, unwrap_exc_or, unwrap_exc_or_default, unwrap_jni, unwrap_jni_verbose,
 };
 pub use self::jni::{get_class_name, get_exception_message};
+
+/// Asserts that given closure panics while executed and the resulting error message contains given
+/// substring.
+#[cfg(test)]
+pub fn assert_panics<F, R>(err_substring: &str, f: F)
```

Comment:
Shall it be a part of the main code?

Reply:
I was unable to come up with scenarios of using this outside the testing scope for the moment 🙂

Reply:
Clippy seems unhappy with integration tests becase of this, thus put it to the main code.

Reply:
Just to clarify: I asked if this function had to be a part of main code because I thought it _already_ was based on its location (exonum-java-binding/core/rust/src/utils/mod.rs, I missed initially `#[cfg(test)]`), and considered it as not needed in the main code (as it is only for testing) just as you did. So, can it be test-only? Shall we suppress clippy or is there a real problem?

Reply:
It turned out that code under the `#[cfg(test)]` annotation is not accessible for the integration tests thus this should be a part of the main code. I looked for other similar helper functions (that should be accessible for both kinds of tests only) but this one seems to be the only one. Is it worth creating a separate module (says, `test_utils`) for it or better to leave it here?